config/config.json.ctmpl

```diff
@@ -5,7 +5,8 @@
 {
 	"site": {
 		"proxyTrust": true,
-		"log": "error"
+		"log": "error",
+		"forceExitAfterMs": 5000
```

Comment:
A ha!

Reply:
Two prong approach: 1. ask nicely to terminate connections and relinquish port and 2. THE HAMMER